src/extension.ts

```diff
@@ -73,7 +73,6 @@ export function activate(context: vscode.ExtensionContext) {
     disposable.push(
         vscode.commands.registerCommand(Commands.CLEAR_OLD_HISTORY, async () => {
             const days = await vscode.window.showInputBox({
-                value: '30',
```

Comment:
:+1: I prefer we display the placeholder rather than hardcode a value of 30 days.